src/swagger/api.yaml

```diff
@@ -2370,7 +2370,7 @@ paths:
           type: string
         - description: Filter expression for groups
           in: query
-          name: filter
+          name: search
```

Comment:
This change needs to go into open API spec before we can update it in the Java SDK. Please create a PR in open API spec repo.

Reply:
https://github.com/okta/okta-management-openapi-spec/pull/47